### PR TITLE
Updating documentation to use redis.asyncio

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -36,7 +36,6 @@ include=[
 [tool.poetry.dependencies]
 python = "^3.7,<=3.11"
 redis = ">=3.5.3,<5.0.0"
-aioredis = "^2.0.0"
```
Comment (YaraslauZhylko) [inline pyproject.toml:39]: `aioredis` got merged with `redis-py` only in version [4.2.0-rc1](https://github.com/redis/redis-py/releases/tag/v4.2.0rc1).

So should we also bump the `redis` dependency version to `4.2.0`? 🤔 
```toml
redis = ">=4.2.0,<5.0.0"
```

Comment (YaraslauZhylko) [reply pyproject.toml:39]: So that people with older `redis` dependencies do no just accidentally bump into errors?

Comment (YaraslauZhylko) [reply pyproject.toml:39]: Or at leas mention this in the docs: https://github.com/redis/redis-om-python/blob/main/docs/getting_started.md#installing-redis-om

Comment (chayim) [reply pyproject.toml:39]: Fair. Documents updated!

**Review by YaraslauZhylko — APPROVED** [review]

**Review by dvora-h — APPROVED** [review]

**Pull request merged**